integration-tests/aqua/examples/abilities.aqua

```diff
@@ -0,0 +1,37 @@
+aqua Main
+
+use DECLARE_CONST, decl_bar from "imports_exports/declare.aqua" as Declare
+
+export handleAb, SomeService
+
+service SomeService("wed"):
+  getStr(s: string) -> string
+
+ability SomeAb:
+  someArrow(s: string) -> string, string
+  str: string
+
+ability SecondAb:
+  arrow(s: string) -> string
+  num: u32
+
+func funcStr(s: string) -> string, string:
+  strInFunc <- SomeService.getStr(Declare.DECLARE_CONST)
+  strInFunc2 <- SomeService.getStr(s)
+  <- strInFunc, strInFunc2
+
+func handleSecAb {SomeAb, SecondAb}() -> string, string, string, u32:
```

Comment:
Is it possible to have several abilities of the same type passed to a single function?